dev.nix

```diff
@@ -99,37 +99,38 @@ pkgs.mkShell {
     # General devtools and test requirements
     pkgs.curl
     pkgs.git
-    (buildDerivation "unzip" "6.0-28" "s9lSnDQ4LMjS5syCmaGFNlBO28KEuRM/++UicEhlBo4=" pkgs.bzip2)
-    (buildDerivation "zip" "3.0-14" "0vsh9c5wfbwsx1r1b5mkfxj5vy1xqv0wbj2i93jysyb7x1c3pq8n" pkgs.zlib)
+    (buildDerivation "unzip" "6.0-29" "YY30d9W3BOQ3z45mBtA68pGtCzPc+5ZNQI940weTOpU=" pkgs.bzip2)
+    (buildDerivation "zip" "3.0-15" "e/y9Af0ZyPdFMzVh/AbEra2fS0FtRfhoXNgfsYf1Wcs=" pkgs.zlib)
 
     # Luau tooling
     pkgs.lune
```

Comment:
The [lune nixos package](https://search.nixos.org/packages?channel=25.05&show=lune&query=lune) seems to be out of date, since we are migrating to v0.10, might want to update this to that version as well, using `fromGithubRelease`.